amd_openvx/openvx/hipvx/vision_kernels.cpp

```diff
@@ -3253,17 +3266,15 @@ Hip_CannySuppThreshold_U8XY_U16_3x3(uint dstWidth, uint dstHeight,
         int goffset = (y - 1) * srcImageStrideInBytes + (x << 3) - 4;
         *((uint2 *)(&lbuf[loffset])) = *((uint2 *)(&pSrcImage[goffset]));
         bool doExtraLoad = false;
-        if (ly < 2) {
-            loffset += 16 * 136;
-            goffset += 16 * srcImageStrideInBytes;
-            doExtraLoad = true;
+        if (ly < 2) {        // }
```

Comment:
this code doesn't look right. the loffset and goffset computation is not happening here